src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -122,8 +124,7 @@ def launch_simulation_from_rq_job(scenario: bytes, settings: Optional[Dict],
         kwargs = {"no_export": True,
                   "seed": settings.get("random_seed", 0)}
 
-        gsy_e.constants.CONNECT_TO_PROFILES_DB = True
-
+        gsy_e.constants.CONNECT_TO_PROFILES_DB = connect_to_profiles_db
```

Comment:
Why do you think we need to make this dynamic? Did you find issues with the `True` value?

Reply:
This test is running the function without a DB. So, it fails to connect to postgres.

Reply:
Thanks for the clarification!
I personally don't like using global constants, so I see your changes as one step in a direction I personally like :D 
At the same time, we're still using and altering a global constant. This means that (AFAIK) the value we set in this test will affect all tests that will be run after this one. If this is confirmed, it's not a good thing because it might have unexpected effects. We should find a way to restore the default value after this test has finished running. Do you confirm this makes sense?

Reply:
Exactly as you said. Tests are failing because global variables are being changed. I'm thinking about a solution ...